Temperature_GIF_Player/Temperature_GIF_Player.ino

```diff
@@ -0,0 +1,335 @@
+// please read credits at the bottom of file
```

Comment:
this file was added to this PR by accident?

Reply:
nah i have 3 guides im workin on